minifi-bootstrap/src/main/java/org/apache/nifi/minifi/bootstrap/util/ConfigTransformer.java

```diff
@@ -704,7 +691,7 @@ protected static void addTextElement(final Element element, final String name, f
     }
 
     public static final String PROPERTIES_FILE_APACHE_2_0_LICENSE =
-            "# Licensed to the Apache Software Foundation (ASF) under one or more\n" +
+            " Licensed to the Apache Software Foundation (ASF) under one or more\n" +
```

Comment:
@phrocker due to the change in the way the props file is written out.

[The store method](https://github.com/apache/nifi-minifi/pull/72/files#diff-a022622bcf128ab992d247f2a8694a0eR263) will prepend a # to the comment string.